src/HomePage.js

```diff
@@ -14,6 +13,7 @@ import RoadStatus from './RoadStatus';
 const HomePage = () => {
     const { getDefaultCountryName } = useContext(CountryStoreContext);
     const [country, setCountry] = useState(undefined);
+    const [result, setResult] = useState({});
```

Comment:
It seems like you are passing `result` and `setResult` to FarmerTable although it has no use of it (just passing it down).
How about using a Context (maybe a new store?)